### PR TITLE
4.0.0

### PR DESCRIPTION
**Pull request by rsanchez15**

### CODE REVIEW

File: vulcanexus.repos [hunk 1]
```diff
@@ -2,100 +2,96 @@ repositories:
...
   eProsima/RMW-Fast-DDS:
     type: git
     url: https://github.com/eProsima/rmw_fastrtps.git
-    version: v3.3.0
+    version: v4.0.0
   eProsima/Shapes-Demo:
     type: git
     url: https://github.com/eProsima/ShapesDemo.git
-    version: v2.14.2
+    version: v3.0.1
```
Comment (Mario-DL) [inline vulcanexus.repos:85]:
```suggestion
    version: v3.1.0
```

**Review by Mario-DL — CHANGES_REQUESTED** [review]
I guess that we will also wait for the releases from `Fast DDS Monitor`  and `Fast DDS Statistics Backend` to be published before merging

**Review by Mario-DL — APPROVED** [review]
LGTM

**Comment by Mario-DL** [comment]
I dont know whether we should add the release notes under `docs/notes/`

File: docs/rst/installation/linux_binary_installation.rst [hunk 2]
```diff
@@ -3,7 +3,7 @@
 Linux binary installation
 =========================
 
-Debian packages for Vulcanexus Iron Imagination are currently available for Ubuntu Jammy.
+Debian packages for Vulcanexus jazzy Imagination are currently available for Ubuntu Jammy.
```
Comment (Mario-DL) [inline docs/rst/installation/linux_binary_installation.rst:6]:
```suggestion
Debian packages for Vulcanexus jazzy Jolo are currently available for Ubuntu Jammy.
```

File: docs/rst/platforms/platforms.rst [hunk 3]
```diff
@@ -20,7 +20,7 @@ Vulcanexus ROS 2 all-in-one tool set, is officially available in the platforms s
       - x86_64 |br|
         arm64
       - Ubuntu Jammy (22.04)
-    * - :ref:`Iron Imagination <notes_iron_latest>` v3
+    * - :ref:`jazzy Imagination <notes_iron_latest>` v3
```
Comment (Mario-DL) [inline docs/rst/platforms/platforms.rst:23]:
```suggestion
    * - :ref:`Jazzy Jolo <notes_jazzy_latest>` v4
```

File: docs/rst/tutorials/core/deployment/ds_wan_tcp/ds_wan_tcp.rst [hunk 4]
```diff
@@ -126,13 +126,13 @@ The expected behavior is that both ``talker`` and ``listener`` are able to conne
 Prerequisites
 -------------
 
-First of all, make sure that Vulcanexus Iron is installed.
+First of all, make sure that Vulcanexus jazzy is installed.
 The docker installation is required for this tutorial (see :ref:`Docker installation <docker_installation>`), together with the `eProsima Vulcanexus docker image <https://hub.docker.com/r/eprosima/vulcanexus>`_.
-The :ref:`Vulcanexus Iron <notes_iron_latest>` image can be downloaded by running:
+The :ref:`Vulcanexus jazzy <notes_iron_latest>` image can be downloaded by running:
```
Comment (Mario-DL) [inline docs/rst/tutorials/core/deployment/ds_wan_tcp/ds_wan_tcp.rst:131]:
```suggestion
The :ref:`Vulcanexus Jazzy <notes_jazzy_latest>` image can be downloaded by running:
```

File: docs/rst/notes/jazzy/notes.rst [hunk 5]
```diff
@@ -0,0 +1,79 @@
+.. _notes_iron_latest:
```
Comment (Mario-DL) [inline docs/rst/notes/jazzy/notes.rst:1]:
```suggestion
.. _notes_jazzy_latest:
```

File: docs/conf.py [hunk 6]
```diff
@@ -93,10 +93,10 @@ def select_css(html_css_dir):
...
 macros = {
-    'DISTRO': 'iron',
-    'DISTRO_TITLE': 'Iron',
-    'DISTRO_TITLE_FULL': 'Iron Irwini',
-    'REPOS_FILE_BRANCH': 'iron',
+    'DISTRO': 'jazzy',
+    'DISTRO_TITLE': 'jazzy',
+    'DISTRO_TITLE_FULL': 'Jazzy Jalisco',
+    'REPOS_FILE_BRANCH': 'jazzy',
     'FASTDDS_BRANCH': 'v2.13.1',
```
Comment (Mario-DL) [inline docs/conf.py:100]: Should this reference be updated ?

File: docs/conf.py [hunk 7]
```diff
@@ -93,10 +93,10 @@ def select_css(html_css_dir):
 project_source_docs_dir = os.path.abspath('{}/rst'.format(script_path))
 
 macros = {
-    'DISTRO': 'iron',
-    'DISTRO_TITLE': 'Iron',
-    'DISTRO_TITLE_FULL': 'Iron Irwini',
-    'REPOS_FILE_BRANCH': 'iron',
+    'DISTRO': 'jazzy',
+    'DISTRO_TITLE': 'jazzy',
+    'DISTRO_TITLE_FULL': 'Jazzy Jalisco',
```
Comment (Mario-DL) [inline docs/conf.py:98]: Should this be `Jazzy Jolo` ?

**Review by Mario-DL — CHANGES_REQUESTED** [review]

File: docs/rst/installation/linux_binary_installation.rst (hunk 2 above)
Comment (Mario-DL) [reply docs/rst/installation/linux_binary_installation.rst:6]: NIT: additionally in capital `Jazzy`

File: docs/rst/platforms/platforms.rst (hunk 3 above)
Comment (rsanchez15) [reply docs/rst/platforms/platforms.rst:23]: Added a new line with jazzy

File: docs/conf.py (hunk 6 above)
Comment (rsanchez15) [reply docs/conf.py:100]: Yes

File: docs/conf.py (hunk 7 above)
Comment (rsanchez15) [reply docs/conf.py:98]: Yes

**Review by Mario-DL — APPROVED** [review]

**Pull request merged**